lte/gateway/c/core/oai/tasks/ngap/include/ngap_client_servicer.h

```diff
@@ -24,8 +24,13 @@ extern "C" {
 #endif
 #include <memory.h>
 #include <string.h>
+#include "lte/protos/oai/s1ap_state.pb.h"
+#include "lte/gateway/c/core/oai/include/map.h"
 
 namespace magma5g {
+typedef magma::map_s<std::string, std::string> map_string_string_t;
+typedef magma::map_s<std::string, magma::lte::oai::S1apImsiMap>
```

Comment:
this should be defined on the `map.h` interface

Reply:
Addressed Review Comment